README.md

```diff
@@ -9,6 +9,16 @@ It reconciles `cluster.cluster.x-k8s.io` objects and makes sure each `Cluster` i
 
 ## Features
 
+### Alertmanager config provisioning
+
+It will look for kubernetes `Secrets` and use them as alertmanager configis if they meet these criteria:
+- a label `observability.giantswarm.io/kind: "alertmanager-config"`
+- an annotation or label `observability.giantswarm.io/tenant` set to the tenant that the alertmanager config should be loaded in.
+
+Current limitations:
+- no support for duplicate configs for the same tenant
```

Comment:
What happens when we get there ? Is this a race then ?

Reply:
The latest one lives

Reply:
<img src="https://media1.giphy.com/media/v1.Y2lkPWJkM2VhNTdlZHVqbXZ3Zm01ZW1pNDlscjdncHhpaDR1Mmh4NWhjZmIwdXljY2l5aSZlcD12MV9naWZzX3NlYXJjaCZjdD1n/edLKLYMlNFPJC/giphy.gif"/>

Reply:
I do not know if we want to support merging in the future or not but anyway, this should be rejected by a kyverno policy imo, I'm not sure how the operator should decide

Reply:
I have no idea how to solve this in a nice way, but that's a nasty case, can you please add in the readme what's happening when duplicate config with same tenant are created.

Reply:
Sure :)

IMO, I would probably add the AlertmanagerConfig CR support in the operator (vendoring the prometheus operator code with a bit of tenancy magic should be possible)

Reply:
Should be better, let me know if you have feedback :)

Reply:
all good :+1: